modules/test/playwright/tests/frontend-data-set-views-web/fields.spec.ts

```diff
@@ -76,7 +78,7 @@ test.describe('Add fields to a view and show them in a fragment', () => {
 		});
 	});
 
-	test('Show Creation Action in fragment', async ({
+	test.skip('Show Creation Action in fragment', async ({
```

Comment:
@chestofwonders This test is broken for me, I get is 404 error to a broken-looking URL. Also, this test should not be in this file, `fields.spec.ts` should hold tests related to Fields. We can fix this in a separate task.

![image](https://github.com/liferay-frontend/liferay-portal/assets/5435511/146b9cd6-dd86-41c8-a898-eee9a887e572)

Reply:
The test is [failing in CI](https://github.com/liferay-frontend/liferay-portal/pull/4002#issuecomment-1956649039), once @juanjofgliferay enabled it.

Reply:
I reverted the `skip` commit, this was an issue on my end.